scripts/in_container/run_test_package_import_all_classes.sh

```diff
@@ -78,7 +78,29 @@ echo
 echo  Importing all classes in Airflow 1.10
 echo
 
+unset PYTHONPATH
 # We need to make sure we are not in the airflow checkout, otherwise it will automatically be added to the
 # import path
 cd /
-python3 /import_all_provider_classes.py
+
+PATHS_TO_SEARCH_PROVIDERS_IN=$(python3 <<EOF 2>/dev/null
+import airflow.providers;
+path=airflow.providers.__path__
+for p in path._path:
+    print(p)
```

Comment:
```suggestion
print(*airflow.providers.__path__)
```

Reply:
That's not good in case of spaces in the name (`print (*` prints array elements in one line separated with space). I have to correct the following few lines where I read the output (with `read -r` to get it working though).